src/usr/local/www/services_dhcp.php

```diff
@@ -764,7 +768,12 @@ function build_pooltable() {
 }
 
 if ($tabscounter == 0) {
-	print_info_box(gettext("The DHCP Server can only be enabled on interfaces configured with a static IPv4 address. This system has none."));
+	if ($have_small_subnet) {
+		$sentence2 = gettext("This system's static IPv4 interfaces have subnet 31 or 32.");
+	} else {
+		$sentence2 = gettext("This system has none.");
+	}
+	print_info_box(gettext("The DHCP Server can only be enabled on interfaces configured with a static IPv4 address with subnet less than 31.") . " " . $sentence2);
```

Comment:
The way CIDR masks and subnet sizes work can confuse some people (larger mask numerically makes a smaller subnet capacity), so I try to avoid referencing it in that way.

Maybe "The DHCP Server requires a static IPv4 subnet large enough to serve addresses to clients." and then in the non-static case the second sentence could be "This interface is not configured with a static IPv4 address."